jdaviz/app.py

```diff
@@ -1570,6 +1571,61 @@ def get_viewer_reference_names(self):
         # Cannot sort because of None
         return [self._viewer_item_by_id(vid).get('reference') for vid in self._viewer_store]
 
+    def update_viewer_reference_name(
+        self, old_reference, new_reference,
+        old_reference_helper_attr=None, update_id=False
+    ):
+        """
+        Update viewer reference names.
+
+        Viewer IDs will not be changed unless `update_id` is True.
+
+        Parameters
+        ----------
+        old_reference : str
+            The viewer reference name to be changed
+        new_reference : str
+            The viewer reference name to use instead of ``old_reference``
+        old_reference_helper_attr : str, optional
+            If one exists, update the helper's attribute `old_reference_helper_attr`
+            with the new viewer reference name. Default is None.
```

Comment:
this is probably fine for now since this isn't intended to be public yet (right?), but if/when we raise this to the helper level, we might want to always handle this automatically.

Reply:
Yeah! Done here: [196f91e](https://github.com/spacetelescope/jdaviz/pull/2338/commits/196f91e3094c73f6f164d20acc45ceefde806988)